porcupine/plugins/run/no_terminal.py

```diff
@@ -477,3 +477,16 @@ def run_command(command: str, cwd: Path) -> None:
     assert runner is not None
     get_vertical_panedwindow().paneconfigure(runner.textwidget, hide=False)
     runner.run_command(cwd, command)
+    # Use previous height as long as it's not less than two lines high,
+    # otherwise set height to two lines high.
```

Comment:
This adds quite a lot of code into an otherwise simple function. Can you move your code into a new function and call it from `run_command()`?